src/client/sandbox/index.js

```diff
@@ -151,7 +151,9 @@ export default class Sandbox extends SandboxBase {
         urlResolver.init(this.document);
 
         // NOTE: Eval Hammerhead code script.
-        this.iframe.on(this.iframe.EVAL_HAMMERHEAD_SCRIPT_EVENT, e => initHammerheadClient(e.iframe.contentWindow, true));
+        this.iframe.on(this.iframe.EVAL_HAMMERHEAD_SCRIPT_EVENT, e => {
+            e.iframe.contentWindow.eval(`(${ initHammerheadClient.toString() })();//# sourceURL=hammerhead.js`);
```

Comment:
Is instruction `//# sourceURL=hammerhead.js` necessary?

Reply:
It needs for debugging purposes.
![image](https://user-images.githubusercontent.com/5373460/48829496-62452f00-ed83-11e8-9f36-70b269d50fb7.png)

Reply:
![image](https://user-images.githubusercontent.com/5373460/48842428-4e122980-eda5-11e8-8d11-191a81e19cb6.png)